soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2784,6 +2784,7 @@ void func_80835F44(GlobalContext* globalCtx, Player* this, s32 item) {
                     Inventory_ChangeEquipment(EQUIP_BOOTS, bootsValue);
                 }
                 Player_SetEquipmentData(globalCtx, this);
+                func_808328EC(this, CUR_EQUIP_VALUE(EQUIP_BOOTS) == 2 ? NA_SE_PL_WALK_HEAVYBOOTS : NA_SE_PL_CHANGE_ARMS);
```

Comment:
It's actually the Iron Boots because the enums are off by one in this usage. Switched to the enum `+ 1` for all three cases within the equip assignments.